infra/cifuzz/actions/run_fuzzers/action.yml

```diff
@@ -2,6 +2,9 @@
 name: 'run-fuzzers'
 description: 'Runs fuzz target binaries for a specified length of time.'
 inputs:
+  project-name:
```

Comment:
@Dor1s @oliverchang Should we call this `oss_fuzz_project_name`? i don't know if/how we'll end up supporting non-OSS-Fuzz users but if they use this it's probably a good idea to be specific I think?

Reply:
makes sense

Reply:
Would also prefer oss_fuzz_project_name to be explicit. This might not even be necessary for non OSS-Fuzz projects right? Also if we do support non OSS-Fuzz users, that seems like a big enough change that any .yml changes are insignificant.

Reply:
Changed to oss-fuzz-project-name.

Reply:
Offline we discussed the fact that if something like "project_name" were needed by non-OSS-Fuzz users we could reuse the variable if it doesn't contain "oss_fuzz". I guess omitting OSS-Fuzz may be confusing if it isn't needed though. Not sure what to do here.